src/components/disclaimer.tsx

```diff
@@ -0,0 +1,23 @@
+import Config from "../config";
+
+export default function DisclaimerBanner() {
+  if (Config.production()) {
+    return (
+      <div className="flex justify-center w-full bg-yellow-400 p-4">
+        <p className="text-sm text-gray-800 text-center">
+          Cursif is currently under heavy <b>developement</b>. 
+          Data loss could occur. 
+          Please use at your own risk. <a href="https://github.com/Code-Society-Lab/cursif" className="underline text-blue-600"><b>Read more</b></a>
+        </p>
+      </div>
+    );
+  }
+  else {
```

Comment:
Doing a else here is problematic. If we are running this in test environment or any other environment (sometime you can have other ones) it will show this. Really checking if we are in dev would be better in my opinion. 

If none of those case (prod or dev), we don't show it at all.